src/cogent3/core/new_sequence.py

```diff
@@ -2113,9 +2113,17 @@ def offset(self, value: int):
     def plus_start(self) -> int:
         """start on plus strand"""
         if self.is_reversed:
-            # self.stop becomes the start, self.stop will be negative
+            # all indices are negative, so the stop becomes the start. To convert
+            # to positive indices, we add the length of the parent sequence to
+            # the stop. Note that when abs(self.step) > 1, we instead calculate
+            # the "true stop", i.e., the index that immediately follows (in the
+            # direction of the step) the last selected index in the slice. We then
+            # add the abs(self.step) to account for the fact that the stop uses
+            # exclusive indexing, and the start is inclusive.
+
             assert self.stop < 0, "expected stop on reverse strand SeqView < 0"
-            start = self.stop + self.parent_len + 1
+            stop = self.stop if self.step == -1 else self.start + len(self) * self.step
```

Comment:
use `self.is_reversed`

Reply:
@GavinHuttley  I'm actually not trying to check if the slice record is reversed, we are already within a conditional where we know we are reversed. I am checking to see if the step is -1, which means we can avoid calculating the modulo-adjusted stop